openerp/addons/base/res/res_users.py

```diff
@@ -564,6 +576,8 @@ def has_group(self, cr, uid, group_ext_id):
                         (SELECT res_id FROM ir_model_data WHERE module=%s AND name=%s)""",
                    (uid, module, ext_id))
         return bool(cr.fetchone())
+    # for a few places explicitly clearing the has_group cache
+    has_group.clear_cache = _has_group.clear_cache
```

Comment:
This works despite the seemingly double declaration for the method, right? (did not check) As in, they're merged by new API magic before being referenced here?